ci/test-coverage.sh

```diff
@@ -32,5 +37,5 @@ else
   codecov -t "${CODECOV_TOKEN}"
 
   annotate --style success --context codecov.io \
-    "CodeCov report: https://codecov.io/github/anza-xyz/agave/commit/${CI_COMMIT:0:9}"
+    "CodeCov report: https://codecov.io/github/anza-xyz/agave/commit/$CI_COMMIT/tree"
```

Comment:
nit: Usually, devs want to directly go to the summary page of their changes. so, `/tree` (all the coverage) can be removed here?

Reply:
fixed: [9ceec9d](https://github.com/anza-xyz/agave/pull/1260/commits/9ceec9d807a4358ac9bf1e45aacc9b7b597fe0c8)